blueocean-dashboard/src/main/js/components/Activity.jsx

```diff
@@ -75,15 +75,15 @@ export class Activity extends Component {
         const head = 'pipelinedetail.activity.header';
 
         const status = t(`${head}.status`, { defaultValue: 'Status' });
-        const build = t(`${head}.build`, { defaultValue: 'Build' });
+        const run = t(`${head}.run`, { defaultValue: 'Run' });
```

Comment:
I assumed it was for missing translations in other languages, but maybe there is a default propsl. Maybe @scherler knows ?